plugin/evm/admin_api.md

```diff
@@ -0,0 +1,524 @@
+---
+title: C-Chain API
+description: "This page is an overview of the C-Chain API associated with AvalancheGo."
+---
+
+<Callout title="Note">
+Ethereum has its own notion of `networkID` and `chainID`. These have no relationship to Avalanche's view of networkID and chainID and are purely internal to the [C-Chain](/docs/quick-start/primary-network#c-chain). On Mainnet, the C-Chain uses `1` and `43114` for these values. On the Fuji Testnet, it uses `1` and `43113` for these values. `networkID` and `chainID` can also be obtained using the `net_version` and `eth_chainId` methods.
+</Callout>
+
+## Ethereum APIs
+
+### Endpoints
+
+#### JSON-RPC Endpoints
+
+To interact with C-Chain via the JSON-RPC endpoint:
+
+```sh
+/ext/bc/C/rpc
+```
+
+To interact with other instances of the EVM via the JSON-RPC endpoint:
+
+```sh
+/ext/bc/blockchainID/rpc
+```
+
+where `blockchainID` is the ID of the blockchain running the EVM.
+
+#### WebSocket Endpoints
+
+<Callout title="info">
+On the [public API node](/docs/tooling/rpc-providers), it only supports C-Chain
+websocket API calls for API methods that don't exist on the C-Chain's HTTP API
+</Callout>
+
+To interact with C-Chain via the websocket endpoint:
+
+```sh
+/ext/bc/C/ws
+```
+
+For example, to interact with the C-Chain's Ethereum APIs via websocket on localhost, you can use:
+
+```sh
+ws://127.0.0.1:9650/ext/bc/C/ws
+```
+
+<Callout title="Tip" icon = {<BadgeCheck className="size-5 text-card" fill="green" />} >
+On localhost, use `ws://`. When using the [Public API](/docs/tooling/rpc-providers) or another
+host that supports encryption, use `wss://`.
+</Callout>
+
+To interact with other instances of the EVM via the websocket endpoint:
+
+```sh
+/ext/bc/blockchainID/ws
+```
+
+where `blockchainID` is the ID of the blockchain running the EVM.
+
+### Standard Ethereum APIs
+
+Avalanche offers an API interface identical to Geth's API except that it only supports the following
+services:
+
+- `web3_`
+- `net_`
+- `eth_`
+- `personal_`
+- `txpool_`
+- `debug_` (note: this is turned off on the public API node.)
+
+You can interact with these services the same exact way you'd interact with Geth (see exceptions below). See the
+[Ethereum Wiki's JSON-RPC Documentation](https://ethereum.org/en/developers/docs/apis/json-rpc/)
+and [Geth's JSON-RPC Documentation](https://geth.ethereum.org/docs/rpc/server)
+for a full description of this API.
+
+<Callout title="info">
+For batched requests on the [public API node](/docs/tooling/rpc-providers) , the maximum
+number of items is 40. We are working on to support a larger batch size.
```

Comment:
consider replacing with :
```
For batched requests on the [public API node](/docs/tooling/rpc-providers), the maximum
request count is 40. We are working on to support a larger batch size.
```
I would suggest removing the `We are working on to support a larger batch size.` unless we have an engineer being tasked with that.

Reply:
added

Reply:
thank you